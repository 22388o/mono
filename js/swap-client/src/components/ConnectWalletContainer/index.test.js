import React from 'react';
import { render, screen, waitFor } from '@testing-library/react';
import '@testing-library/jest-dom'; // For the "toBeInTheDocument" matcher
import { ConnectWalletContainer } from './index';

describe('ConnectWalletContainer component test', () => {
  it('renders component', async () => {

    const {container} = render(
      <ConnectWalletContainer />
    );

    await waitFor(() => {
      expect(container).toMatchSnapshot();
    });
  });
});
