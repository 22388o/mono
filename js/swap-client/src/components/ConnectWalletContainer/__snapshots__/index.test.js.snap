// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConnectWalletContainer component test renders component 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    style="display: none;"
  >
    <div
      class="MuiStack-root css-nen11g-MuiStack-root"
    >
      <div
        class="MuiStack-root css-m69qwo-MuiStack-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          Connect Wallet
        </p>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="KeyboardDoubleArrowRightIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M6.41 6 5 7.41 9.58 12 5 16.59 6.41 18l6-6z"
            />
            <path
              d="m13 6-1.41 1.41L16.17 12l-4.58 4.59L13 18l6-6z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiStack-root css-nen11g-MuiStack-root"
      >
        <div
          class="MuiStack-root css-m69qwo-MuiStack-root"
          id="metamask-connect-btn"
        >
          <img
            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTMJyiAlYerxZx6dsXo5Pzv5gLdCrHKJ-5gnEs9RTGQ70RNCHoICMQ8&usqp=CAE&s"
            style="border-radius: 5px;"
            width="32"
          />
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Metamask
          </p>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="MuiStack-root css-3y1137-MuiStack-root"
        >
          <div
            class="MuiStack-root css-m69qwo-MuiStack-root"
            id="alby-connect-btn"
          >
            <img
              src="https://seeklogo.com/images/W/walletconnect-logo-EE83B50C97-seeklogo.com.png"
              style="border-radius: 5px;"
              width="32"
            />
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Alby
            </p>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="MuiStack-root css-3y1137-MuiStack-root"
        >
          <div
            class="MuiStack-root css-m69qwo-MuiStack-root"
            id="unisat-connect-btn"
          >
            <img
              src="https://seeklogo.com/images/W/walletconnect-logo-EE83B50C97-seeklogo.com.png"
              style="border-radius: 5px;"
              width="32"
            />
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Unisat
            </p>
          </div>
        </div>
        <hr
          class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
        />
        <div
          class="MuiStack-root css-3y1137-MuiStack-root"
        >
          <div
            class="MuiStack-root css-m69qwo-MuiStack-root"
            id="xverse-connect-btn"
          >
            <img
              src="https://seeklogo.com/images/W/walletconnect-logo-EE83B50C97-seeklogo.com.png"
              style="border-radius: 5px;"
              width="32"
            />
            <p
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
            >
              Xverse
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
